app/helpers/ops_helper/settings_label_tag_mapping_helper.rb

```diff
@@ -0,0 +1,55 @@
+module OpsHelper::SettingsLabelTagMappingHelper
+  private
+
+  def settings_label_tab_mapping_list(warning, mappings)
+    {
+      :headers => settings_label_tab_mapping_headers,
+      :rows    => settings_label_tab_mapping_rows(mappings),
+      :warning => warning,
+    }
+  end
+
+  def settings_label_tab_mapping_headers
+    [
+      {:header => _("Resource Entity"), :key => 'resource_entity'},
+      {:header => _("Resource Label"), :key => 'resource_label'},
+      {:header => _("Tag Category"), :key => 'tag_category'},
+      {:header => _("Actions"), :key => 'actions'},
```

Comment:
Super-minor, but this might read clearer aligned like a table:

```suggestion
      {:header => _("Resource Entity"), :key => 'resource_entity'},
      {:header => _("Resource Label") , :key => 'resource_label'},
      {:header => _("Tag Category"),    :key => 'tag_category'},
      {:header => _("Actions"),         :key => 'actions'},
```